src/components/Editable.js

```diff
@@ -160,9 +213,18 @@ export const Editable = connect()(({ isEditing, thoughtsRanked, contextChain, sh
         }
       }
     }}
+    onKeyDown={(e) => {
+      // if enter pressed run and flush the latest event
+      if (e.keyCode === 13) {
+        throttledChangeRef.current.flush()
+      }
+    }}
```

Comment:
There are a large number of [`shortcuts`](https://github.com/cybersemics/em/tree/2a1e91fdd9a4268f5cc8a81477b23971be83b7f5/src/shortcuts) that may require a flush. How do you propose we handle all of them? Should we store the throttledChangeRef in state so that other handlers can access it?

Reply:
Storing `throttleChangeRef` which is a ref value in state would be an antipattern.  Instead we can just so something like this:

```javascript
      // if shortcut pressed run and flush the latest event
      if (shortcutKeyIndex[hashKeyDown(e)]) throttledChangeRef.current.flush()
```